example/helpers.js

```diff
@@ -12,11 +12,14 @@ const getValue = path(['target', 'value'])
 const isNotEmpty = a => a.trim().length > 0
 const hasCapitalLetter = a => /[A-Z]/.test(a)
 const isGreaterThan = curry((len, a) => (a > len))
+const isLessThan = curry((len, a) => (a < len))
 const isLengthGreaterThan = len => compose(isGreaterThan(len), prop('length'))
+const isLengthLessThan = len => compose(isLessThan(len), prop('length'))
```

Comment:
additional validator added for demonstration of something validated but not required.